.gitignore

```diff
@@ -117,6 +117,7 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+.huaxia_env
```

Comment:
what is this for?  is it necessary?

Reply:
Thanks for the feedback! Was about to hide this.